lib/optypes/receipt.go

```diff
@@ -0,0 +1,89 @@
+// This file copies Receipt type from ethereum-optimism/op-geth/core/types/receipt.go
```

Comment:
Couple things.

Would have imported, but op-geth is just a fork of geth that changes minimal things. So:
- we get an error that we import with one name but the module has another
- geth versions won't match

Figured copying a little is better.

As for comment at the top, would have added a copyright. But the [op file itself](https://github.com/ethereum-optimism/op-geth/blob/optimism/core/types/receipt.go) has go-ethereum copyright. Any thoughts? Is this comment fine?

Reply:
I don't think it matters that much, above is fine.